docs/containers-registries.conf.d.5.md

```diff
@@ -15,7 +15,8 @@ By default, the directory is located at `/etc/containers/registries.conf.d`.
 
 Once the main configuration at `/etc/containers/registries.conf` is loaded, the
 files in `/etc/containers/registries.conf.d` are loaded in alpha-numerical
-order. Specified fields in a config will overwrite any previous setting.  Note
+order. Then the conf files in `$HOME/.config/containers/registries.conf.d` are loaded in alpha-numerical order, if they exist.
+Specified fields in a conf file will overwrite any previous setting.  Note
```

Comment:
(Eventually this will need updating after the semantics is settled.)

Reply:
@mtrmac  not sure how to describe the logic in the manpage